src/components/H5PElement/h5pApi.ts

```diff
@@ -6,12 +6,12 @@
  *
  */
 
-import { Author } from '../../interfaces';
+import { IAuthor } from '@ndla/types-draft-api';
```

Comment:
Dette er litt skummelt synes jeg. Har vi noen tanker om å lage en `@ndla/types-common`? Kjipt å ha en mismatch på `IAuthor` mellom to API's, for eksempel.

Reply:
I fremtiden ja, men først må vi lage common mappe og flytte på ting i backenden.